ios/Classes/Module/FacebookModule.m

```diff
@@ -183,7 +183,7 @@ - (void)logPurchase:(NSArray<id> *_Nonnull)purchase
 
 - (void)logRegistrationCompleted:(NSString *_Nonnull)registrationMethod
 {
-  NSDictionary *params = @{ FBSDKAppEventParameterNameRegistrationMethod : registrationMethod };
+  NSDictionary *params = [[NSDictionary alloc] initWithObjectsAndKeys:registrationMethod, FBSDKAppEventParameterNameRegistrationMethod, nil];
```

Comment:
I don't see a logic difference here, except that you use the "old" non-literal NSDictionary constructor now.

Reply:
The difference is in the tracked event on Facebook. On the facebook manager I had many warning about the missing registration completed event. I tried to init the dictionary using its constructor and now the completed registration events are correctly tracked.